src/components/Header/JumbotronContent/AirCityHeaderContent/NeedlePointer.js

```diff
@@ -32,7 +32,7 @@ function NeedlePointer({ classes, hidden, measurement }) {
   return (
     <g transform="translate(450,370)" className={className}>
       <path
-        transform={`rotate(${rotate}) scale(1.2)`}
+        transform={`rotate(${rotate - 4.0}) scale(1.2)`}
```

Comment:
Why are we changing this rotation. For example, for the below, it should be directly on 5 but it's slightly off:

![Screenshot from 2019-04-30 08-59-42](https://user-images.githubusercontent.com/1779590/56943060-7a855b00-6b26-11e9-9875-23fbae902c37.png)

Reply:
@kilemensi Noticed that most of the readings were not accurately represented. So for example, the review request you made on the gauge circle reads 7.3 but shows a reading on the gauge for 4.8 is value. So added a  - 4 to the value for accuracy

Reply:
`rotate` was fine @nerdlet. Since we're changing `height`, we may need to just adjust `translate` y-axis value.